### PR TITLE
NE-741/e2e/Azureprovider

### PR DESCRIPTION
**Pull request by Connect2naga**
adding azure test execution in ci main & 4.10 release

### CODE REVIEW

File: ci-operator/jobs/openshift/external-dns-operator/openshift-external-dns-operator-release-4.10-presubmits.yaml [hunk 1]
```diff
@@ -50,15 +50,15 @@ presubmits:
...
+    - ^release-4\.10$
+    - ^release-4\.10-
     cluster: build03
     context: ci/prow/e2e-azure-operator
     decorate: true
     labels:
       ci.openshift.io/generator: prowgen
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-openshift-external-dns-operator-main-e2e-azure-operator
+    name: pull-ci-openshift-external-dns-operator-4.10-e2e-azure-operator
```
Comment (lmzuccarelli) [inline ci-operator/jobs/openshift/external-dns-operator/openshift-external-dns-operator-release-4.10-presubmits.yaml:61]: Good catch

**Comment by Connect2naga** [comment]
this PR will be failed until merge the PR https://github.com/openshift/external-dns-operator/pull/70

**Pull request merged**